scripts/loki_transform.py

```diff
@@ -207,7 +207,8 @@ def convert(
     if not remove_code_trafo:
         remove_code_trafo = RemoveCodeTransformation(
             remove_marked_regions=True, remove_dead_code=False,
-            call_names=('ABOR1', 'DR_HOOK'), intrinsic_names=('WRITE(NULOUT',)
+            call_names=scheduler.config.default.get('call_names', None) or ('ABOR1', 'DR_HOOK'),
```

Comment:
@awnawab I think you can just configure the transformation itself with all of it's constructor arguments in the config file as something like (untested):
```
transformations.RemoveCodeTransformation
  module = "loki.transform"
transformations.RemoveCodeTransformation.options
  remove_marked_regions = true
  remove_dead_code = false
  call_names =['dr_hook', 'abor1']
  intrinsic_names = ['write(nulout']
```
This way, no custom assumptions or behaviour are added to the `convert` step, which is current direction of travel (removing all the custom logic). Ultimately the plan was to remove all of this and configure the entire pipeline in config, but I've no full-scale example of this yet. Does this make sense?

Reply:
Agreed - transformation-specific configuration should not be added to `defaults`

Reply:
Aah, I had completely missed the addition of this functionality! Thanks for pointing this out, of course the above suggestion works and I'll close this PR 👍